apps/web/src/app/(default)/dashbord/[slug]/page.tsx

```diff
@@ -57,6 +58,11 @@ export default async function EventDashboard({ params }: Props) {
     },
   });
 
+  const getCsvData = await getHappeningCsvData(happening.id);
+  if (!getCsvData) {
+    return notFound();
+  }
```

Comment:
Kan man ikke heller bytte ut querien over med denne? De henter jo så å si det samme.

```suggestion
  const happening = await getHappeningCsvData(happening.id);
  
  if (!happening) {
    return notFound();
  }
```

Reply:
Er dette bare et navnebytte? Der er en annen variabel som allerede heter happening i funksjonen over, EventDashboard

Reply:
Også, var greia med formatteringen bare CRLF og LF?

Reply:
Så å si bare et navnebytte, ha. Du bare fjerner fetchen over og bytter ut med med csv.

Var noe med LF og CRLF med formateringen, ja.